### DISCUSSION TITLE
Syntax Error

### DISCUSSION THREAD

**Discussion by gxcora** [facelessuser/soupsieve#236] [Q&A] [answered]
Hi:
I'm getting this syntax error with beautifulsoup4 (4.10.0) and soupsieve (2.3.1). Note sure if this is related to recent changes you made to util.py

`Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/soupsieve/__init__.py", line 29, in <module>
    from . import css_parser as cp

  File "/usr/local/lib/python3.5/dist-packages/soupsieve/css_parser.py", line 4, in <module>
    from . import util

  File "/usr/local/lib/python3.5/dist-packages/soupsieve/util.py", line 59
    f"'{func.__name__}' is deprecated. {message}",
                                            
SyntaxError: invalid syntax`

**Comment by facelessuser** [comment, marked as answer]
Yeah, so we changed that line of code to use "f" strings. Keep in mind that you seem to be on Python 3.5, but we currently support Python 3.6+. Python 3.6 is reaching "end of life" very soon as well: https://endoflife.date/python. With that said, we probably won't immediately drop Python 3.6 until the next release with meaningful changes.

**Reply by facelessuser** [reply]
Also, it should be noted that dropping doesn't mean "won't work", it just means we no longer "test to ensure it works". We obviously haven't supported Python 3.5 for a bit now, but only recently it has broken as we start to adopt newer Python syntax for things.

**Comment by gxcora** [comment]
thanks
